airline/views.py

```diff
@@ -14,10 +15,19 @@ def create(self, request, *args, **kwargs):
         serializer = self.get_serializer(data=request.data)
         serializer.is_valid(raise_exception=True)
         self.perform_create(serializer)
-        headers = self.get_success_headers(serializer.data)
+        headers = self.get_succsdess_headers(serializer.data)
         return Response(serializer.data, status=status.HTTP_201_CREATED, headers=headers)
 
     def retrieve(self, request, *args, **kwargs):
         instance = self.get_object()
         serializer = self.get_serializer(instance)
         return Response(serializer.data)
+    
+class AirlineListAPIView(generics.ListAPIView):
```

Comment:
As we are working with ViewSets this logic can be implemented in the AirlineViewSet class without having to create an APIView. 

Right now with the current main I added the `django_filters` package. Take a look at it and mostly review the config for using it: https://django-filter.readthedocs.io/en/stable/guide/usage.html 

If you get the updated main branch you have this example here: 

1. This is how you declare the filter: https://github.com/cm0639-group3/web-tickets-be/blob/main/airplane/filters.py
2. You register it inside of the viewset: https://github.com/cm0639-group3/web-tickets-be/blob/main/airplane/views.py
3. And here in the tests you can see how it should behave: https://github.com/cm0639-group3/web-tickets-be/blob/main/airplane/tests.py#L69